charts/ingress-nginx/README.md

```diff
@@ -252,11 +252,11 @@ Kubernetes: `>=1.20.0-0`
 | controller.admissionWebhooks.networkPolicyEnabled | bool | `false` |  |
 | controller.admissionWebhooks.objectSelector | object | `{}` |  |
 | controller.admissionWebhooks.patch.enabled | bool | `true` |  |
-| controller.admissionWebhooks.patch.image.digest | string | `"sha256:549e71a6ca248c5abd51cdb73dbc3083df62cf92ed5e6147c780e30f7e007a47"` |  |
+| controller.admissionWebhooks.patch.image.digest | string | `"sha256:39c5b2e3310dc4264d638ad28d9d1d96c4cbb2b2dcfb52368fe4e3c63f61e10f"` |  |
 | controller.admissionWebhooks.patch.image.image | string | `"ingress-nginx/kube-webhook-certgen"` |  |
 | controller.admissionWebhooks.patch.image.pullPolicy | string | `"IfNotPresent"` |  |
 | controller.admissionWebhooks.patch.image.registry | string | `"registry.k8s.io"` |  |
-| controller.admissionWebhooks.patch.image.tag | string | `"v1.3.0"` |  |
+| controller.admissionWebhooks.patch.image.tag | string | `"v20220916-gd32f8c343"` |  |
```

Comment:
do we want this type of tag to be in the released chart?

Reply:
Hi, @cpanato @rikatz ,
I will change it back if you say so. I will have to re-promote that image using a PR in k8s.io repo.

The reason I did this was that the tag convention `1.3.0` seems not based on any real data. For example this time, I could have set the tag as `1.3.2` but that is just a near-future highly likely possibility that we will release app version `1.3.2` . At the time of promoting the image, there is no app version `1.3.2` in existence. Also, we may not rebuild this image until a alpine bump or a golang bump, so potentially several future releases of the controller (e.g. 1.3.2, 1.3.3.....1.4.x) will all carry a certgen image with tag `1.3.2`, and that too would be a mismatch with the app-version.

With the tag set to this convention, the date of building the image and some kind of a build-id is a real inference of the tag.

But even @rikatz had told me earlier about the choice of tag string so there must be some reason why you are raising this question on this type of tag. So I look forward to advise on what is the reason that a aesthetically appealing but less real-data-based tag like `1.3.x` was the previous convention and preferred.

Whatever you and @rikatz finally decide, I will make changes accordingly.